diff -up media-driver-intel-media-24.4.4/media_common/linux/common/os/mos_defs_specific.h.orig media-driver-intel-media-24.4.4/media_common/linux/common/os/mos_defs_specific.h
--- media-driver-intel-media-24.4.4/media_common/linux/common/os/mos_defs_specific.h.orig	2024-11-28 08:58:40.000000000 +0100
+++ media-driver-intel-media-24.4.4/media_common/linux/common/os/mos_defs_specific.h	2025-01-30 23:31:05.345189790 +0100
@@ -29,6 +29,7 @@
 
 #include <pthread.h>
 #include <semaphore.h>
+#include <cstdint>
 #include <string>
 
 typedef pthread_mutex_t         MOS_MUTEX, *PMOS_MUTEX;         //!< mutex pointer
